File: app/src/main/java/com/castelijns/mmdemo/user_detail/UserDetailContract.java
package com.castelijns.mmdemo.user_detail;

import com.castelijns.mmdemo.app.BasePresenter;
import com.castelijns.mmdemo.app.BaseView;

public interface UserDetailContract {

    interface View extends BaseView {
        void showNavigationTo(String lat, String lon);
    }

    interface Presenter extends BasePresenter {
        void onDirectionsClicked();
    }
}

File: app/src/main/kotlin/com/castelijns/mmdemo/user_detail/UserDetailContract.kt
package com.castelijns.mmdemo.user_detail

import com.castelijns.mmdemo.app.BasePresenter
import com.castelijns.mmdemo.app.BaseView

interface UserDetailContract {

    interface View : BaseView {
        fun showNavigationTo(lat: String, lon: String)
    }

    interface Presenter : BasePresenter {
        fun onDirectionsClicked()
    }
}
